cypress/integration/login_test.js

```diff
@@ -52,5 +52,32 @@ describe('Login tests', () => {
 
       cy.wait('@httpsRequest');
     });
+
+    it('use custom Astarte URLs when configured to do so', function () {
+      cy.fixture('config/custom_urls').then((userConfig) => {
+        cy.route('/user-config/config.json', userConfig);
+        cy.route('https://api.example.com/custom-appengine/health', '').as(
+          'customAppengineRequest',
```

Comment:
this is not just a "custom-request". Maybe call it `appEngineHealthRequest`?
also carefull on the componenet case, it's AppEngine, not Appengine

Reply:
Fixed it, thanks